### PR TITLE
Release 3.5.0

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-layout?_sp.spa=pr-release-3.5.0](https://host.nxt.blackbaud.com/skyux-layout?_sp.spa=pr-release-3.5.0)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 3.5.0 (2020-01-27)
+
+- Added the inline delete component. [#56](https://github.com/blackbaud/skyux-layout/pull/56)
+- Added logic to the card component to handle the inline delete component. [#56](https://github.com/blackbaud/skyux-layout/pull/56)
```
Comment (Blackbaud-AlexKingman) [inline CHANGELOG.md]: `to handle`? Could we be more specific? Maybe something about allowing the inline delete component to appear inside the cards template? Not sure how to word this.

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-layout?_sp.spa=pr-release-3.5.0](https://host.nxt.blackbaud.com/skyux-layout?_sp.spa=pr-release-3.5.0)

File: CHANGELOG.md (hunk 1 above)
Comment (blackbaud-johnly) [reply CHANGELOG.md]: How about `Added logic to the card component to allow it to contain the inline delete component.`? Does that sound better to you, @Blackbaud-AlexKingman?

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: Hmmmm. I think I like the second one better @blackbaud-johnly

Comment (blackbaud-johnly) [reply CHANGELOG.md]: Sounds good. ... I'll make the change.

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-layout?_sp.spa=pr-release-3.5.0](https://host.nxt.blackbaud.com/skyux-layout?_sp.spa=pr-release-3.5.0)

**Pull request merged**